test/jit/test_upgraders.py

```diff
@@ -81,3 +92,112 @@ def f():
         upgraders_dump_after_remove_test = torch._C._dump_upgraders_map()
         self.assertTrue("a" not in upgraders_dump_after_remove_test)
         self.assertTrue("c" not in upgraders_dump_after_remove_test)
+
+    def test_aten_div_tensor_at_3(self):
+        model_path = pytorch_test_dir + "/jit/fixtures/test_versioned_div_tensor_v3.pt"
+        loaded_model = torch.jit.load(model_path)
+        # there are 3 aten::div in this model
+        # And the upgrader for aten::div uses two
+        # div's because of if/else branch
+        FileCheck().check("prim::If").run(loaded_model.graph)
+        FileCheck().check_count("aten::div", 6).run(loaded_model.graph)
+
+        buffer = io.BytesIO()
+        torch.jit.save(loaded_model, buffer)
+        buffer.seek(0)
+        version = self._load_model_version(loaded_model)
+        self.assertTrue(version == 4)
+        loaded_model_twice = torch.jit.load(buffer)
+        # we check by its' code because graph variable names
+        # can be different every time
+        self.assertEqual(loaded_model.code, loaded_model_twice.code)
+
+    def test_aten_test_serialization(self):
```

Comment:
It is a bit hard to read and maintain such test cases. For examle, I do not know what is inside `_test_serialization_subcmul_v2.pt` to tell whether the test is correct. Could we add a TODO for later to automatically generate such test cases from structured data with ample comment for readability?

Reply:
sounds good. I added a TODO at the top of the test file marking this